proxy/nginx.conf.template

```diff
@@ -308,3 +308,11 @@ server {
   auth_basic_user_file  "/etc/nginx/.htpasswd";
   <% end %>
 }
+
+<% if ENV.key?("SERVICE_DOMAIN_SV") && ENV.key?("SERVICE_DOMAIN") %>
+server {
+    listen 8080;
+    server_name <%= ENV["SERVICE_DOMAIN_SV"] %>;
+    rewrite ^ https://<%= ENV["SERVICE_DOMAIN"] %>$uri?lang=sv redirect;
+}
```

Comment:
IMO we should log correctly here, too. Also the security-headers like "Strict-Transport-Security".